build/js/live-editor.output_pjs.js

```diff
@@ -1731,10 +1731,10 @@ var BabyHint = {
         "curve": 8,
         "cursor": [0, 1, 2, 3],
         "endShape": [0, 1],
-        "dist": 4,
-        "fill": [1, 3, 4],
+        "dist": [4, 6],
+        "fill": [1, 2, 3, 4],
         "filter": [1, 2],
-        "get": [2, 3, 4, 5],
+        "get": [0, 2, 3, 4, 5],
```

Comment:
I don't use get() much myself but I know it's popular in the community. Is the performance ok when you call get() on an entire image?

Reply:
Smaller `get` snapshots are faster, but [some community members want to be able to invoke `get` without passing in any arguments](https://khanacademy.zendesk.com/hc/en-us/community/posts/115005702387-Documentation-for-get-x-y-w-h-).